x-pack/plugins/security/server/authorization/privileges/feature_privilege_builder/cases.ts

```diff
@@ -10,17 +10,20 @@ import { uniq } from 'lodash';
 import type { FeatureKibanaPrivileges, KibanaFeature } from '../../../../../features/server';
 import { BaseFeaturePrivilegeBuilder } from './feature_privilege_builder';
 
+export type CasesSupportedOperations = typeof allOperations[number];
```

Comment:
I'm not going to admit how long it took me to figure out how to get this typed correctly haha

Here's an explanation of the [number] syntax: https://stackoverflow.com/questions/59541521/whats-the-meaning-of-typeof-arraynumber-in-typescript

![image](https://user-images.githubusercontent.com/56361221/153271940-cccf43ff-b903-4828-880c-b88bcbd63c94.png)